index.html

```diff
@@ -57,6 +60,164 @@ <h1 id="primary-text">
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="work-card">
+                    <div class="bottom-area">
+                        <div class="top">
+                            <h1 class="text">
+                                Multi-post stories<br>
+                                Gain+Glory
+                            </h1>
+                        </div>
+
+                        <ul class="tech">
+                            <li class="tech-card">
+                                <p>Ruby on Rails</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="work-card">
+                    <div class="bottom-area">
+                        <div class="top">
+                            <h1 class="text">
+                                Multi-post stories<br>
+                                Gain+Glory
+                            </h1>
+                        </div>
+                        <ul class="tech">
+                            <li class="tech-card">
+                                <p>Ruby on Rails</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section id="about-me">
+            <div id="ab-me-img"></div>
+            <div id="ab-me-1-img"></div>
+            <div id="ab-me-desktop-img"></div>
+            <div id="ab-me-desktop-1-img"></div>
+
```

Comment:
- [ ]  I would suggest that if an image is not part of the content (aesthetic reasons) use CSS `background-image` instead of `img` tag (decorations are always background images, not img tags). [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

Reply:
@shella12 I don't understand, there are no img tags there already :(

Reply:
> @shella12 I don't understand, there are no img tags there already :(

Hi @Movgto sorry for the confusion what I meant was don't use div tags to place images just simply put them in the background of the section since you can use multiple images in the background and you can position them each using `background-postion`. right?